### PR TITLE
add tracker to sidebar button

### PR DESCRIPTION
**Pull request by jezebelbaal**
### Description

This PR adds tracker to calendar sidebar button

### Context & Notes

[https://buffer.atlassian.net/browse/PUB-3766](https://buffer.atlassian.net/browse/PUB-3766)

### Best Practices Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] I have tested different user stories.
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [ ] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [ ] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [ ] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

### CODE REVIEW

File: packages/profile-sidebar/index.js [hunk 1]
```diff
@@ -68,6 +70,12 @@ export default hot(
         dispatch(campaignsPage.goTo());
       },
       onCalendarButtonClick: () => {
+        const metadata = {
+          source: 'sidebar',
+          calendarVersion: 'new',
+          view: 'week',
```
Comment (anafilipadealmeida) [inline packages/profile-sidebar/index.js:76]: I'm feeling a bit doubtful about this one. I understand why adding a view "week", as the calendar selected by the default is the week. At the same time, the action itself is not done with the intent of opening a week view (unlike the queue buttons). The other reason why I'd probably not pass this one is that it's hard to maintain - if we change the calendar to open the month by default, this tracking event wouldn't be automatically updated. What do you think @jezebelbaal ?

Comment (jezebelbaal) [reply packages/profile-sidebar/index.js:76]: Hi Ana! From your ideas, I think that could be a tag such as default-week, default-month, for instance. So we know that the customer did not choose that option in an intentional way and we could separate them from intentional clicks. Does that make sense? What do you think, @jwinternheimer ?

Comment (anafilipadealmeida) [reply packages/profile-sidebar/index.js:76]: I'd personally not overcomplicate it and just not pass it @jezebelbaal . We had asked Julian to make it not required already thinking about this scenario. We'll still be able to analyze all the data without it 😄

Comment (jezebelbaal) [reply packages/profile-sidebar/index.js]: See your point Ana, just removed and we can assume it. Thanks for sharing : )

**Review by anafilipadealmeida — APPROVED** [review]

**Pull request merged**